.github/workflows/ci.yaml

```diff
@@ -112,15 +112,6 @@ jobs:
             test/integration \
             -m ''
 
-      # Run regression tests
-      - name: Regression tests
-        run: |
-          pytest \
-            --cov=autosubmit --cov-config=.coveragerc \
-            --cov-report=xml:test/coverage.xml --cov-append \
-            test/regression \
-            -m ''
-
```

Comment:
This was run in integration and in regression build jobs. I've removed from integration, and now it only runs in the regression job.